device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/TransportUtils.java

```diff
@@ -8,6 +8,7 @@
 public class TransportUtils
 {
     public static String IOTHUB_API_VERSION = "2019-10-01";
+    public static String PNP_IOTHUB_API_VERSION = "2020-05-31-preview";
```

Comment:
Please add a detail comment as to why are we adding preview API here

Reply:
added